packages/admin/admin/src/theme/componentsTheme/MuiPickersInputBase.ts

```diff
@@ -0,0 +1,85 @@
+import { iconButtonClasses, inputAdornmentClasses } from "@mui/material";
+import { type Components, type Theme } from "@mui/material/styles";
+import { pickersInputBaseClasses, pickersInputClasses, pickersSectionListClasses } from "@mui/x-date-pickers";
+import type {} from "@mui/x-date-pickers/themeAugmentation";
+
+import { mergeOverrideStyles } from "../utils/mergeOverrideStyles";
+
+export const getMuiPickersInputBase = (
+    component: Components["MuiPickersInputBase"],
+    { palette, spacing }: Theme,
+): Components["MuiPickersInputBase"] => ({
+    ...component,
+    styleOverrides: mergeOverrideStyles(component?.styleOverrides, {
+        root: {
+            border: `1px solid ${palette.grey[100]}`,
+            borderRadius: 2,
+            backgroundColor: "#fff",
```

Comment:
Optional: we could use `palette.background.paper` or `palette.background.default` here to support dark mode.

Reply:
Generally a good idea, in this case however I explicitly want the `PickersInputBase` to look and behave the same as the normal `InputBase`. So if we do it, we should do this in all `InputBase` components and in a separate PR.